kubectl/kubectl.go

```diff
@@ -0,0 +1,44 @@
+package kubectl
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"strings"
+)
+
+// Path is the path of the kubectl binary
+var Path = "kubectl"
+
+// Runner is an interface to wrap kubectl convenience methods
+type Runner interface {
+	ClusterInfo() ([]byte, error)
```

Comment:
hmm, at some point I'll look into why `golint` gives nothing. thanks for the comments!